src/Components/ProvisioningWizard/steps/AccountCustomizations/aws.js

```diff
@@ -67,6 +68,37 @@ const AccountCustomizationsAWS = ({ setStepValidated, architecture, composeID })
       >
         <RegionsSelect composeID={composeID} />
       </FormGroup>
+      <FormGroup
+        label="Select template"
+        fieldId="aws-select-template"
+        labelIcon={
+          <Popover
+            bodyContent={
+              <span>
+                Launch templates contains the configuration information to launch an instance, For further information and for creating launch
```

Comment:
@lzap, here is the help text for it, I also added a link for more details

Reply:
Great thanks.